docs/DeveloperGuide.adoc

```diff
@@ -415,6 +415,67 @@ Certain properties of the application can be controlled (e.g App name, logging l
 
 This section describes the details on how the volunteer management features are implemented.
 
+// tag::exportcert[]
+==== Export Volunteer Certificate
+The `exportcert` command enables the volunteer manager to export a PDF document (certificate) detailing a volunteer's involvement with the organisation. This is only possible when in the 'volunteer' context. The information included in the certificate are as follows:
+
+* Organisation's logo
+* Volunteer Name and NRIC
+* Date of PDF certificate creation
+* List of events involved in
+* Hours of service contributed
+
+This is a sample of the exported PDF certificate:
+
+image::VolunteerCert.png[width="600"]
```

Comment:
Is it possible to add a border around the sample cert? The white blends into the background of the documentation. Makes it uneasy on the eyes.